tests/failing

```diff
@@ -14,6 +14,11 @@ tests/simple/var.uplc
 # ifThenElse needs to be composed with force
 # We need to improve error report to cope with `uplc` output.
 tests/simple/ifThenElse-no-force.uplc
+# Non-iterative application is not supported
```

Comment:
please change this to something like "non-iterative application coming soon" or something similar so that we don't forget about this. Otherwise, it looks good!

Reply:
I will fix this in a subsequent commit.